### DISCUSSION TITLE
Multiple Accessories

### DISCUSSION THREAD

**Discussion by Ursis-Horibilis** [RavenSystem/esp-homekit-devices#1851] [MEPLHAA Scripts]
Trying to add a switch acc & Temp probe acc.  Below is my config code.  They both are added to homekit but the temp probe is configured as a switch not a temperature.  What am I doing wrong?

{
  "c": { "l": 13, "b": [{ "g": 0, "t": 5 }], "n": "Pool Heater Fire SW" },
  "a": [{
    "t": 1,
    "d": 3600,
    "s": 5,
    "0": { "r": [{ "g": 12, "v": 0 }] },
    "1": { "r": [{ "g": 12, "v": 1 }] },
    "b": [{ "g": 0 }]
  },
	{
	"a": [
		{"t": 22, "g": 14, "n": 3}
		]
	}
	]  
}

**Comment by paulzhdanov** [comment]
Try this:

{
"c": { "l": 13, "b": [{ "g": 0, "t": 5 }], "n": "Pool Heater Fire SW" },
"a": [{
	"t": 1,
	"d": 3600,
	"s": 5,
	"0": { "r": [{ "g": 12, "v": 0 }] },
	"1": { "r": [{ "g": 12, "v": 1 }] },
	"b": [{ "g": 0 }]
},{
	"t": 22, 
	"g": 14, 
	"n": 3
}]
}

**Reply by Ursis-Horibilis** [reply]
That worked.  Thank you.
